include/TS3Data.h

```diff
@@ -21,11 +23,16 @@ class TS3Data : public TGRSIDetectorData  {
     std::vector<UShort_t>  fS3_SectorDetector;
     std::vector<UShort_t>  fS3_SectorNumber;
     std::vector<TFragment> fS3_SectorFragment;
+    
+   static bool fIsSet;      //!
```

Comment:
We have found that bool's in root take an entire byte. I will leave this for now, but you should make a Short so you have effectively 8 bools you can save for the price of one (checkout griffin crystal and pileup to see what I mean).